doc/install/build-ceph.rst

```diff
@@ -38,17 +38,7 @@ repository and execute the following::
     cd ceph
     ./do_cmake.sh
     cd build
-    make
-
-.. note:: By default do_cmake.sh will build a debug version of ceph that may
-   perform up to 5 times slower with certain workloads. Pass 
-   '-DCMAKE_BUILD_TYPE=RelWithDebInfo' to do_cmake.sh if you would like to
-   build a release version of the ceph executables instead.
-
-.. topic:: Hyperthreading
-
-	You can use ``make -j`` to execute multiple jobs depending upon your system. For 
-	example, ``make -j4`` for a dual core processor may build faster.
+    ninja
```

Comment:
Is this note outdated? If not, it's good to have it.

Reply:
Yes, hyperthreading part of the note is outdated. I have added a note to refer `README`.